src/app/teams/teams.component.html

```diff
@@ -36,13 +36,13 @@ <h3>
       <ng-container matColumnDef="visitLog.lastVisit">
         <mat-header-cell *matHeaderCellDef mat-sort-header i18n>Most recent visit</mat-header-cell>
         <mat-cell *matCellDef="let element">
-          {{element.visitLog.lastVisit | date: 'medium'}}
+          {{(element.visitLog.lastVisit | date: 'medium') || 'N/A'}}
         </mat-cell>
       </ng-container>
       <ng-container matColumnDef="visitLog.visitCount">
         <mat-header-cell *matHeaderCellDef mat-sort-header i18n>Total visits from last 30 days</mat-header-cell>
         <mat-cell *matCellDef="let element">
-          {{element.visitLog.visitCount}}
+          {{element.visitLog.visitCount || 'N/A'}}
```

Comment:
This should not be changed to `N/A`.  If the visit count is 0, we should show 0.